indigo_app/static/javascript/indigo/views/annotations.js

```diff
@@ -443,8 +442,11 @@
       this.threadViews = [];
       this.visibleThreads = [];
       this.counts.set({threads: 0});
-      this.threads.forEach(t => this.makeView(t));
-      this.renderAnnotations();
+      //Wait for page to load implicitly
+      window.setTimeout(() => {
```

Comment:
This makes me nervous because it looks like we have a race condition, and you never know if 500ms will be enough. Let's talk through it.

Reply:
Agreed.